Ui example does not build with feature `conrod`

```
> cargo run --features conrod --example ui
[snip]
   Compiling kiss3d v0.21.0 (D:\dev\experiments\kiss3d)
error[E0433]: failed to resolve: use of undeclared type or module `HashMap`
  --> src\window\window.rs:52:26
   |
52 |             texture_ids: HashMap::new()
   |                          ^^^^^^^ use of undeclared type or module `HashMap`

error[E0412]: cannot find type `HashMap` in this scope
  --> src\window\window.rs:43:18
   |
43 |     texture_ids: HashMap<String, conrod::image::Id>
   |                  ^^^^^^^ not found in this scope
help: possible candidates are found in other modules, you can import them into scope
   |
5  | use conrod::image::HashMap;
   |
5  | use std::collections::HashMap;
   |
5  | use std::collections::hash_map::HashMap;
   |
[snip]
error: aborting due to 2 previous errors

Some errors have detailed explanations: E0412, E0433.
For more information about an error, try `rustc --explain E0412`.
error: Could not compile `kiss3d`.
```

I had to make the following changes to get it to compile:
```diff
diff --git a/examples/ui.rs b/examples/ui.rs
index dd641698..44c48cb2 100644
--- a/examples/ui.rs
+++ b/examples/ui.rs
@@ -12,6 +12,14 @@ use kiss3d::conrod;
 use kiss3d::conrod::color::Color;
 #[cfg(feature = "conrod")]
 use kiss3d::conrod::position::Positionable;
+#[cfg(feature = "conrod")]
+use kiss3d::window::Window;
+#[cfg(feature = "conrod")]
+use kiss3d::light::Light;
+#[cfg(feature = "conrod")]
+use kiss3d::conrod::widget_ids;
+#[cfg(feature = "conrod")]
+use std::path::Path;
 
 #[cfg(not(feature = "conrod"))]
 fn main() {
diff --git a/src/window/window.rs b/src/window/window.rs
index aa3fc509..fb81b4cd 100644
--- a/src/window/window.rs
+++ b/src/window/window.rs
@@ -32,6 +32,8 @@ use window::{Canvas, State};
 use image::{GenericImage, Pixel};
 #[cfg(feature = "conrod")]
 use renderer::ConrodRenderer;
+#[cfg(feature = "conrod")]
+use std::collections::HashMap;
 
 static DEFAULT_WIDTH: u32 = 800u32;
 static DEFAULT_HEIGHT: u32 = 600u32;
```
I have not tested if any other examples have the same issue.

--- a/examples/ui.rs
+++ b/examples/ui.rs
@@ -12,6 +12,14 @@ use kiss3d::conrod;
 use kiss3d::conrod::color::Color;
 #[cfg(feature = "conrod")]
 use kiss3d::conrod::position::Positionable;
+#[cfg(feature = "conrod")]
+use kiss3d::conrod::widget_ids;
+#[cfg(feature = "conrod")]
+use kiss3d::light::Light;
+#[cfg(feature = "conrod")]
+use kiss3d::window::Window;
+#[cfg(feature = "conrod")]
+use std::path::Path;
 
 #[cfg(not(feature = "conrod"))]
 fn main() {

--- a/src/window/window.rs
+++ b/src/window/window.rs
@@ -32,6 +32,8 @@ use window::{Canvas, State};
 use image::{GenericImage, Pixel};
 #[cfg(feature = "conrod")]
 use renderer::ConrodRenderer;
+#[cfg(feature = "conrod")]
+use std::collections::HashMap;
 
 static DEFAULT_WIDTH: u32 = 800u32;
 static DEFAULT_HEIGHT: u32 = 600u32;

